dome/install_windows.md

```diff
@@ -79,4 +79,4 @@ conda uninstall libignition-<package_name> --channel conda-forge
 
 ## Troubleshooting
 
-See [Troubleshooting](troubleshooting.md#windows)
+See [Troubleshooting](/docs/dome/troubleshooting#windows)
```

Comment:
Same as Citadel. [Dome troubleshooting page](https://ignitionrobotics.org/docs/dome/troubleshooting) doesn't have a windows section either. Remove `#windows` if you could please.